examples/data-sources/b1ddi_dns_hosts/data-source.tf

```diff
@@ -16,3 +15,9 @@ data "b1ddi_dns_hosts" "dns_host_by_name" {
     name = "dns_host_name"
   }
 }
+
+data "b1ddi_dns_hosts" "all_dns_host_by_tags" {
```

Comment:
Inconsistencies with other types:
1. No comment,
2. Plural should be used for `all_dns_host_by_tags`, e.g. `all_dns_hosts_by_tags`.

Reply:
Comments were added to all objects with plurals . 
Plurals were also fixed in the object name